### ISSUE TITLE
Derivate compiler output and duplicate files in source repo

### ISSUE THREAD

**Issue by eepstein** [Modular-Network/ethereum-libraries#87]
## What is it?

You've got the outputs from the truffle compile in the (source code) repo.  That's an anti-pattern.

### Problem

Keep source code in the repo.
Along with notes and settings and your package.json if needed so a dev can recreate the outputs.

Put your outputs in a release package, like npm or similar (truffle offers a packaging tool).

Don't keep duplicates of your source files in the same repo.  Refactor.  DRY (don't repeat yourself), etc.

Basically, this is not the recommended way to use a code repo.

### Solution

Delete derivate files and re-commit and push.

### Steps to implement

1. Provide either a numbered list
2. That that provides steps to implement   

* Or a bullet list of items
* If order doesn't matter
